de.metas.business/src/main/java-legacy/org/compiere/model/CalloutOrder.java

```diff
@@ -314,7 +314,11 @@ public String bPartnerLocation(final ICalloutField calloutField)
 
 		if (!orderBL.setBillLocation(order))
 		{
-			throw new BPartnerNoBillToAddressException(bpartner);
+			final String localizedMessage = new BPartnerNoBillToAddressException(bpartner).getLocalizedMessage();
+			calloutField.fireDataStatusEEvent(
```

Comment:
wtf, i had no clue we can do this :-)

Reply:
yeah i blame teo as well, he's the one who showed this trickery :D